modules/enabling-entra-workload-id-existing-cluster.adoc

```diff
@@ -0,0 +1,247 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-entra-workload-id-existing-cluster"]
+= Enabling {entra-first} on an existing cluster
+
+If you did not configure your {azure-first} {product-title} cluster to use {entra-first} during installation, you can enable this authentication method on an existing cluster.
+
+[IMPORTANT]
+====
+The process to enable {entra-short} on an existing cluster is disruptive and takes a significant amount of time.
+Before proceeding, observe the following considerations:
+
+* Read the following steps and ensure that you understand and accept the time requirement.
```

Comment:
Just wondering if it might be worthwhile to mention an estimate of the time requirement here? You call out it below for specific steps, but someone would have to look through and add them up. I see 4 different things that can take 15 mins, plus a few others. Just wondering if an estimate of over an hour (or longer) or something would be helpful to say here?